.github/ISSUE_TEMPLATE/pull-request.md

```diff
@@ -0,0 +1,30 @@
+## Description
```

Comment:
So, I like the bug report template, it's helpful! but I think I'm ok with dramatically cutting (or removing) the pull request one. I think the people that tend to submit PRs have a decent understanding of the questions being asked here, so I don't think this friction is needed.

Reply:
done!